File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int product = 1;
        int sum = 0;

        for (int i = 0; i < num_list.length; i++) {
            product *= num_list[i];
            sum += num_list[i];
        }
        
        int sumTotal = sum * sum;

        if (product < sumTotal) {
            return 1;
        } else {
            return 0;
        } 
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    answer = 0

    num1 = 1
    num2 = 0

    for num in num_list:
        num1 *= num

    for num in num_list:
        num2 += num

    num2 = num2 ** 2

    if num1 < num2:
        answer = 1
    else:
        answer = 0

    return answer